packages/python/accern_data/accern_data.py

```diff
@@ -23,26 +25,86 @@
     write_json,
 )
 
-FiltersType = TypedDict("FiltersType", {
-    "provider_id": Optional[str],
-    "entity_name": Optional[str],
-    "event": Optional[str],
-    "entity_ticker": Optional[str],
-    "entity_accern_id": Optional[str],
-}, total=False)
-
-
-FilterField = {
+ExcludedFilterField = Literal[
+    "crawled_at",
+    "format",
+    "harvested_at",
+    "published_at",
+    "token",
+]
```

Comment:
date

--> add _all_ special arguments

Reply:
why published and crawled at?

Reply:
do we need to make use of them (in our code) in future?